03-setup-ingress-controller/guides/wildcard_certificates.md

```diff
@@ -0,0 +1,682 @@
+- The example provided in this tutorial is using `starter-kit.online` domain name (and naming convention). Please make sure to use your own domain name and naming convention.
+
+Next, create the `Certificate` resource in your `DOKS` cluster (please pick only one option, depending on the installed `Ingress Controller`):
+
+- `Ambassador Edge Stack` ingress:
+
+  ```shell
+  kubectl apply -f 03-setup-ingress-controller/assets/manifests/cert-manager-ambassador-wcard-certificate.yaml
+  ```
+
+- `Nginx` ingress:
+
+  ```shell
+  kubectl apply -f 03-setup-ingress-controller/assets/manifests/cert-manager-nginx-wcard-certificate.yaml
+  ```
+
+Verify certificate status (please pick only one option, depending on the installed `Ingress Controller`):
+
+- `Ambassador Edge Stack` ingress:
+
+  ```shell
+  kubectl get certificate starter-kit.online -n ambassador
+  ```
+
+- `Nginx` ingress:
+
+  ```shell
+  kubectl get certificate starter-kit.online -n ingress-nginx
+  ```
+
+The output looks similar to (notice the `READY` column value - should be `True`, and the `SECRET` name):
+
+```text
+NAME                 READY   SECRET               AGE
+starter-kit.online   True    starter-kit.online   3m8s
+```
+
+**Notes:**
+
+- Please bear in mind that it can take a `few minutes` for the process to complete.
+- If the `Certificate` object reports a `not ready state` for some reason, then you can fetch the logs from the `Cert-Manager Controller` Pod and see why the `Certificate` failed:
+
+    ```shell
+    kubectl logs -l app=cert-manager,app.kubernetes.io/component=controller -n cert-manager
+    ```
+
+Finally, you can verify the `Kubernetes` secret which contains your `TLS` certificate (please pick only one option, depending on the installed `Ingress Controller`):
+
+- `Ambasador Edge Stack` ingress:
+
+  ```shell
+  kubectl describe secret starter-kit.online -n ambassador
+  ```
+
+  The output looks similar to (notice that it contains the `wildcard` certificate `private` and `public` keys):
+
+  ```text
+  Name:         starter-kit.online
+  Namespace:    ambassador
+  Labels:       <none>
+  Annotations:  cert-manager.io/alt-names: *.starter-kit.online,starter-kit.online
+                cert-manager.io/certificate-name: starter-kit.online
+                cert-manager.io/common-name: *.starter-kit.online
+                cert-manager.io/ip-sans: 
+                cert-manager.io/issuer-group: cert-manager.io
+                cert-manager.io/issuer-kind: Issuer
+                cert-manager.io/issuer-name: letsencrypt-ambassador-wcard
+                cert-manager.io/uri-sans: 
+
+  Type:  kubernetes.io/tls
+
+  Data
+  ====
+  tls.crt:  5632 bytes
+  tls.key:  1679 bytes
+  ```
+
+- `Nginx` ingress:
+
+  ```shell
+  kubectl describe secret starter-kit.online -n ingress-nginx
```

Comment:
The command should be:
kubectl describe secret starter-kit.online -n backend

Reply:
Should be fixed in the latest commit. Thanks.